client/components/room/Video.jsx

```diff
@@ -301,7 +307,58 @@ class Video extends React.Component {
     }
   }
 
-  // Added 'muted' here to help reduce audio feedback but not entirely sure it will work
+  requestTurn() {
+    const turnURL = 'https://numb.viagenie.ca?username=andy.yeo@gmail.com&key=hackreactor';
```

Comment:
Same as above, all this is testing. This branch was meant to be only for the share/copy link.